en_US/mqtt/auto_subscribe/auto_subscribe.md

```diff
@@ -0,0 +1,93 @@
+# MQTT
+
+## Auto Subscribe
+
+When a device is successfully connected to EMQX, no additional subscription requests (SUBSCRIBE messages) need to be sent and EMQX completes the pre-defined subscription relationship for the device. This feature is improved from the MQTT proxy subscription feature in EMQX version 4.
```

Comment:
```suggestion
With Auto Subscription enabled, after a client is successfully connected to EMQX, EMQX completes the pre-defined subscription for the client. So that the client does not need to send `SUBSCRIBE` requests. 

Prior to version 5, this feature is also known as "proxy subscription".
```